.build/parent-pom.xml

```diff
@@ -0,0 +1,1020 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>apache</artifactId>
+    <groupId>org.apache</groupId>
+    <version>22</version>
+  </parent>
+  <groupId>org.apache.cassandra</groupId>
+  <artifactId>cassandra-parent</artifactId>
+  <version>4.1-alpha2-SNAPSHOT</version>
```

Comment:
I am worried about maintaining this; maybe we should keep the `write_pom` task and use that to update the version?  so these act more like a template?

Reply:
good catch @dcapwell 
given the variables from build.xml that need to be re-used in the pom files, there would still need to be a generation step before install/publish…

Reply:
spoke with Abe in Slack and he is going to try out `copy` and `filter` tasks to see if we can turn these files into a template, this also has a side benefit of shrinking the change as we add back the write pom task , task ordering, and tmp files.